test-forms/index.json

```diff
@@ -488,7 +489,7 @@
   { "html": "www_neurology_org_login.html", "generated": true, "title": "User account | American Academy of Neurology Journals", "expectedFailures": ["username OR emailAddress"], "comment": "rank: 9960" },
   { "html": "ibs_johnsoncontrols_com_login.html", "generated": true, "title": "Johnson Controls Login", "expectedFailures": ["username OR emailAddress"], "comment": "rank: 9995" },
   { "html": "bitwarden_com_account-locked.html" },
-  { "html": "duckduckgo_cloudsave-passphrase.html" },
+  { "html": "duckduckgo_cloudsave-passphrase.html", "expectedFailures": ["password"] },
```

Comment:
If we decided that the algorithm has the correct UX, we should change the `data-manual-scoring`, not adding an exception here.